src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -553,7 +553,7 @@ public void Initialize()
             }
 
             options.SchemaVersion = SchemaUpgrader.CurrentVersion.ServerVersion;
-            options.SchemaUpgrader = SchemaUpgrader.Upgrader(SchemaUpgrader.StorageType.Server, null, null);
+            options.SchemaUpgrader = SchemaUpgrader.Upgrader(SchemaUpgrader.StorageType.Server, null, null, this);
```

Comment:
do we need to pass ServerStore?

Reply:
Not anymore, I used a short term context instead.